app/dao/tokens_dao.py

```diff
@@ -0,0 +1,22 @@
+from app import db
+from app.models import Token
+
+
+def save_token_model(token, update_dict={}):
```

Comment:
This function can probably have the update part removed, we should allow updating of the token, just creating/renewing.